app/views/users/new.html.erb

```diff
@@ -1,3 +1,17 @@
 <% provide :title, t(".title") %>
 <h1><%= t ".title" %></h1>
-<p><%= t ".desc" %></p>
+<div class="row">
+  <div class="mx-auto col-md-6 col-md-offset-3">
+    <%= form_with model: @user, local: true do |f| %>
```

Comment:
Sử dụng form tag để sign in up đi em, in params trong 2 trường hợp đó lên đây.
So sánh form_with, form_for và form_tag

Reply:
- em có so sánh ở bên trên rồi, form_tag thì truyền url còn form_for truyền model

Reply:
In params của 3 dạng lên đây em,

Reply:
![image](https://user-images.githubusercontent.com/68627044/89005854-a8f87400-d32f-11ea-8472-9211bc01f880.png)

Reply:
Chương trình còn chạy đúng không em?

Reply:
không vì nó sẽ k gửi lên param được require là user